src/librustc_target/abi/mod.rs

```diff
@@ -11,7 +11,7 @@
 pub use self::Integer::*;
 pub use self::Primitive::*;
 
-use spec::Target;
+use spec::{Target, AddrSpaceIdx, };
```

Comment:
Nit: you have these `, ` in a few places, you should remove them.

Reply:
Ok